<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accueil</class>
 <widget class="QMainWindow" name="Accueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Images/noir1.jpg);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>23</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextEdit" name="chat">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 75 italic 9pt &quot;Raavi&quot;;
font: 8pt &quot;Comic Sans MS&quot;;
 border-style: outset;
    border-width:3px;
    border-radius: 8px;
border-color:rgb(28, 81, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(106, 173, 255);
selection-background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 75 italic 9pt &quot;Raavi&quot;;
font: 8pt &quot;Comic Sans MS&quot;;
 border-style: outset;
    border-width:3px;
    border-radius: 8px;
border-color:rgb(28, 81, 255);</string>
           </property>
           <property name="text">
            <string> Send  </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="saisie">
         <property name="styleSheet">
          <string notr="true">border-color:rgb(28, 81, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> border-style: outset;
    border-width:6px;
    border-radius:20px;
border-color:rgb(255, 30, 233);
font: 75 italic 11pt &quot;Mangal&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVe">
    <property name="title">
     <string>Fichier</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuParametres">
    <property name="title">
     <string>Parametres</string>
    </property>
    <addaction name="actionEnregistrer_la_discussion"/>
    <addaction name="actionChanger_police"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuVe"/>
   <addaction name="menuParametres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnregistrer_la_discussion">
   <property name="text">
    <string>Enregistrer la discussion</string>
   </property>
  </action>
  <action name="actionChanger_police">
   <property name="text">
    <string>Changer couleur</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
